src/common/CommonHeader.css

```
@@ -0,0 +1,45 @@
+
+.common-header__menu ion-list {
+  padding: 4px 0;
+  background: transparent;
+}
+
+.common-header__menu ion-item {
+  --background: transparent;
+  --color: var(--text-strong, #191f28);
+  --min-height: 46px;
+  --padding-start: 16px;
+  font-size: 15px;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)